ddcommon/src/rate_limiter.rs

```diff
@@ -54,7 +54,15 @@ fn now() -> u64 {
             tv_nsec: 0,
         };
         unsafe { libc::clock_gettime(libc::CLOCK_MONOTONIC, &mut ts) };
-        (ts.tv_sec * TIME_PER_SECOND + ts.tv_nsec) as u64
+        // tv_sec is i32 on 32bit architecture
```

Comment:
I would add link to the original issue 

```suggestion
        // tv_sec is i32 on 32bit architecture
        // https://sourceware.org/bugzilla/show_bug.cgi?id=16437
```